docs/production-install.rst

```diff
@@ -48,37 +45,32 @@ Then the repository definition:
   enabled=1
   EOF
 
+Install the Socorro repository.
+::
+  sudo rpm -ivh https://s3-us-west-2.amazonaws.com/org.mozilla.crash-stats.packages-public/el/7/noarch/socorro-public-repo-1-1.el7.centos.noarch.rpm
+
 Now you can actually install the packages:
 ::
   sudo yum install postgresql93-server postgresql93-plperl \
     postgresql93-contrib java-1.7.0-openjdk python-virtualenv \
-    rabbitmq-server elasticsearch httpd mod_wsgi memcached daemonize
-
-Enable Apache on startup:
-::
-  sudo service httpd start
-  sudo chkconfig httpd on
+    rabbitmq-server elasticsearch httpd mod_wsgi memcached socorro
```

Comment:
Are you sure about `httpd` and `mod_wsgi` here given that we're now using Nginx and uwsgi?  Maybe `s/httpd\smod_wsgi/nginx/;` or so?